assets/css/edd-admin.css

```diff
@@ -1921,7 +1921,7 @@ body.download_page_edd-reports {
 }
 
 .edd-chip {
-	font-size: 7px;
+	font-size: 10px;
 	text-transform: uppercase;
 	font-weight: bold;
 	line-height: 8px;
```

Comment:
Let's set this to `line-height: 1;` so it always matches the current font size, which will fix ensure it stays snug to the text but also stop any cascading line heights from applying.